lib/lor/lib/router/layer.lua

```diff
@@ -5,7 +5,7 @@ local pairs = pairs
 local ipairs = ipairs
 local type = type
 local setmetatable = setmetatable
-local ostime = os.time
+local ostime = ngx.time
```

Comment:
os.time is only used for `math.randomseed` here. this small change will breaks most test cases for now.
maybe it's better to redefine `ostime` variable later.